spark3/src/main/java/org/apache/iceberg/spark/Spark3Util.java

```diff
@@ -320,7 +320,7 @@ public static Distribution buildRequiredDistribution(org.apache.iceberg.Table ta
     }
   }
 
-  private static DistributionMode getDistributionMode(org.apache.iceberg.Table table) {
+  public static DistributionMode getDistributionMode(org.apache.iceberg.Table table) {
```

Comment:
Minor: If this is going to be public, we should fix the name. `get` doesn't add any value. How about `distributionModeFor` instead?

Reply:
I don't like adding `get` either but the method is doing something. I think `distributionModeFor` may be a bit better, I'll change.

Reply:
Fixed.